google/cloud/firestore_v1/base_document.py

```diff
@@ -188,12 +188,10 @@ def create(self, document_data: dict) -> NoReturn:
     def set(self, document_data: dict, merge: bool = False) -> NoReturn:
         raise NotImplementedError
 
-    def update(
-        self, field_updates: dict, option: _helpers.WriteOption = None
-    ) -> NoReturn:
+    def update(self, field_updates: dict, option=None) -> NoReturn:
```

Comment:
Why are we going back on the annotations here?
```suggestion
    def update(
        self, field_updates: dict, option: _helpers.WriteOption = None
    ) -> NoReturn:
```